packages/frontend/src/design-system/tokens/utils.css.ts

```diff
@@ -42,3 +44,27 @@ export const flexColumnCenter = style([
 export const boxShadow = style({
   boxShadow: "0 3px 10px rgba(0,0,0,0.1), 0 3px 3px rgba(0,0,0,0.05)",
 });
+
+export const scrollBarHidden = style({
+  overflow: "scroll",
+  msOverflowStyle: "none",
+  scrollbarWidth: "none",
+  selectors: {
+    "&::-webkit-scrollbar": {
+      display: "none",
+    },
+  },
```

Comment:
```suggestion
  "::-webkit-scrollbar": {
    display: "none",
  },
```

가상 요소는 selectors 사용하지 않아도 되는 걸로 알고 있어요!

Reply:
감사합니다!!

a3dcdbef2b841ada077d40904e68f24e12d03192✨